extensions/amp-facebook/1.0/README.md

```diff
@@ -0,0 +1,438 @@
+for Facebook videos embeds the player of the video, and adds the accompanying
+post card with it. Setting `data-embed-as="post"` ignores the caption card. This
+is done to make sure we are zooming in on videos correctly.
+
+The `comment` value embeds a single comment (or reply to a comment) on a post.
+This is not to be confused with
+[`amp-facebook-comments`](https://amp.dev/documentation/examples/components/amp-facebook-comments/).
+
+Check out the documentation for differences between
+[post embeds](https://developers.facebook.com/docs/plugins/embedded-posts),
+[video embeds](https://developers.facebook.com/docs/plugins/embedded-video-player),
+and [comment embeds](https://developers.facebook.com/docs/plugins/embedded-comments).
+
+##### `includeCommentParent`
+
+The value is either `true` or `false`. The default is `false`.
+
+When you are embedding a comment reply, you can optionally also include the
+parent comment of the reply.
+
+##### `allowfullscreen`
+
+The value is either set or omitted. The default is omitted (no fullscreen).
+
+When embedding a video, set this value to allow for a fullscreen experience.
+
+##### `alignCenter`
+
+The value is either `true` or `false`. The default is `false`.
+
+For posts and videos, having this attribute set to true would align the
+post/video container to center.
+
+##### `locale` (optional)
+
+By default, the locale is set to user's system language; however, you can
+specify a locale as well.
+
+For details on strings accepted here please visit the
+[Facebook API Localization page](https://developers.facebook.com/docs/internationalization).
+
+##### title (optional)
+
+Define a `title` attribute for the component to propagate to the underlying `<iframe>` element. The default value is `"Facebook"`.
+
+##### `numposts` (optional)
+
+The number of comments to show. Currently Facebook SDK limits this to max 100.
+For details, see the
+[Facebook comments documentation](https://developers.facebook.com/docs/plugins/comments).
+
+##### `orderBy` (optional)
+
+The order to use when displaying comments. For details, see the
+[Facebook comments documentation](https://developers.facebook.com/docs/plugins/comments).
+
+##### `action` (optional)
+
+The verb to display on the button. Can be either `like` or `recommend`. The
+default is `like`.
+
+##### `kdSite` (optional)
+
+This attribute is also known as `data-kid_directed_site` in the Facebook SDK.
+If your web site or online service, or a portion of your service, is directed to
+children under 13 you must enable this attribute. The default is `false`.
+
+##### `dataLayout` (optional)
+
+Selects one of the different layouts that are available for the plugin. Can be
+one of `standard`, `button_count`, `button` or `box_count`. The default is
+`standard`.
+
+##### `ref` (optional)
+
+A label for tracking referrals which must be less than 50 characters and can
+contain alphanumeric characters and some punctuation.
```

Comment:
```suggestion
##### `refLabel` (optional)

A label for tracking referrals which must be less than 50 characters and can
contain alphanumeric characters and some punctuation.
```
This is called `refLabel` for the preact component because `ref` is a keyword